content/posts/blog/xmpp-summit-21.md

```diff
@@ -0,0 +1,21 @@
+---
+title: XMPP Summit 21
+date: 2017-02-27 18:50
+author: Guus
+blog_id: blog
+---
+
+This year, the [XMPP Standards Foundation](https://xmpp.org) again gathered in force to attend the summit, that traditionally precedes the FOSDEM event in Brussels, Belgium. Barely fitting in the (rather sizable) room that was made available to us by Cisco, the XSF members had a fruitful two-day meeting. 
+
+The attending members, skillfully herded by Kev, addressed an impressive number of topics, including:
+
+* BIND2, improving the data exchange that occurs when an XMPP entity initially connects to a server.
+* MIX, the XEP that intends to be a replacement for the existing MUC protocol, bringing an up-to-date feature set and better extensibility.
+* Addressing the annoyance of instant-messaging-based spam.
+* The application of existing, or to be developed XMPP standards, to facilitate the growing world of Internet-of-Things.
+* The XSF intention to take part in this years [Google Summer of Code](https://summerofcode.withgoogle.com/).
+* Improvements to the existing end-to-end encryption, as defined in the [OMEMO XEP](https://xmpp.org/extensions/xep-0384.html).
+
+The full minutes are made available in the [XSF wiki](https://wiki.xmpp.org/web/Summit_21_minutes). If any of these topics (or others for that matter) interest you, we'd love to hear from you. Please find us at [xmpp.org](https://xmpp.org), use the mailinglists to contact us, or find us in one of the dedicated MUCs.
+
+Apart from the discussions, the summit proved to be an excellent opportunity for some of the newer members amongst us, myself included, to make acquaintance with the others. I, for one, am grateful to have been there, and am looking forward to the next meetup!
```

Comment:
I obviously have no problem with it. I'll leave it in, unless someone else objects, okay?